instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -50,6 +50,24 @@
 # pylint: disable=too-many-public-methods
 
 
+def normalize_arguments(doc_type, body=None):
+    if major_version == 7:
+        return {"document": body} if body else {}
+    else:
+        return (
+            {"body": body, "doc_type": doc_type}
+            if body
+            else {"doc_type": doc_type}
+        )
+
+
+def ElasticsearchFactory(*args, **kwargs):
+    client = Elasticsearch(*args, **kwargs)
+    if major_version == 7:
```

Comment:
For context elasticsearch 8 will need to mock another attribute

Reply:
Elasticsearch 8 or 7?

Reply:
This is for 7, for 8 need to mock another attribute